README.md

```diff
@@ -73,9 +73,15 @@ $ cd src/backend/conf && touch dev.conf
 
 ```bash
 DBName = wayne
-# MySQL address,if MySQL is started via docker-compose,
+# MySQL connection config, default its container name for its in the same network with wayne,
+# if you run MySQL via docker-compose and you didn't change the mysql's container name, just
+# keep it default.
+# You can also run "docker network inspect wayne_default"(replace wayne_default to the real 
+# docker network MySQL container use) to get the real IP for wayne_mysql_1, and replace 
+# MySQL docker name(wayne_mysql_1) to the real IP. Its more flexible when you want customize
+# your environment for wayne running. For example "DBTns = tcp(172.17.0.2:3306)"
 # Cannot be accessed directly through 127.0.0.1 under Mac OS.Please change to actual IP.
-DBTns = tcp(127.0.0.1:3306)
+DBTns = tcp(wayne_mysql_1:3306)
```

Comment:
why we need this change

Reply:
```suggestion
DBTns = tcp(wayne_mysql_1:3306)
```

because `wayne_mysql_1` is the default contianer name for mysql when you execute `docker-compose up -d mysql` and set it to `wayne_mysql_1` is convenient.

Reply:
Mac can't visit mysql via wayne_mysql_1. Please add links to docker-compose file, rather than use wayne_mysql_1.
``` yaml
  wayne:
    image: 360cloud/wayne:latest
    command: /opt/wayne/backend apiserver
    environment:
      GOPATH: "/go"
    volumes:
      - ./src/backend/conf:/opt/wayne/conf:z
    networks:
      - default
    expose:
      - "8080"
    ports:
      - "8080:8080"
    links:
      - mysql:mysql
```

Reply:
Also update your comments at the same time

Reply:
> Mac can visit mysql via mysql.Refer to #158 ,
> Also update your comments at the same time

I tried on my mac
``` shell
docker exec -it wayne_wayne_1 ping mysql
PING mysql (172.19.0.2) 56(84) bytes of data.
64 bytes from wayne_mysql_1.wayne_default (172.19.0.2): icmp_seq=1 ttl=64 time=0.191 ms
64 bytes from wayne_mysql_1.wayne_default (172.19.0.2): icmp_seq=2 ttl=64 time=0.119 ms
64 bytes from wayne_mysql_1.wayne_default (172.19.0.2): icmp_seq=3 ttl=64 time=0.107 ms

docker exec -it wayne_wayne_1 ping wayne_mysql_1
PING wayne_mysql_1 (172.19.0.2) 56(84) bytes of data.
64 bytes from wayne_mysql_1.wayne_default (172.19.0.2): icmp_seq=1 ttl=64 time=0.121 ms
64 bytes from wayne_mysql_1.wayne_default (172.19.0.2): icmp_seq=2 ttl=64 time=0.142 ms
64 bytes from wayne_mysql_1.wayne_default (172.19.0.2): icmp_seq=3 ttl=64 time=0.116 ms
64 bytes from wayne_mysql_1.wayne_default (172.19.0.2): icmp_seq=4 ttl=64 time=0.141 ms
```

mysql is the service name, wayne_mysql_1 is the container name, which one you think its better to use?